### PR TITLE
Feature/home container

### PR DESCRIPTION
**Pull request by W3stside**
committed similar messages twice - second commit should read:
`edited Home component, container + story`

### CODE REVIEW

File: src/api/initialization.ts [hunk 1]
```diff
@@ -36,7 +36,7 @@ const contractArtifacts = [
 ].map(name => require(`../../build/contracts/${name}.json`))
 
 class DutchExchangeInit {
-  contracts: Object | any
+  contracts: object | any
```
Comment (Velenir) [inline src/api/initialization.ts:39]: `object | any === any`
Not much point in it. Feel free to add something like 
```
interface Contracts{ DutchExchange: any, DutchExchangeFactory: any }
```
to types

**Review by Velenir — APPROVED** [review]

File: src/selectors/blockchain.ts [hunk 2]
```diff
@@ -22,6 +22,8 @@ export const findDefaultProvider = (state: State) => {
   })
 }
 
+export const getActiveProvider = (state: State): Providers[string] => selector(state).activeProvider
```
Comment (Velenir) [inline src/selectors/blockchain.ts]: `Providers[string]` is `any`
To grab from Providers values you need `Providers[keyof Providers]`
Another way would be to specify possible providers:
```
type ProviderName = 'METAMASK' | 'MIST'
interface Providers {
 [P in ProviderName]: any
}
```
then `(state: State): ProviderName`
also we'll need Provider interface at some point

**Review by Velenir — APPROVED** [review]

File: src/api/initialization.ts (hunk 1 above)
Comment (W3stside) [reply src/api/initialization.ts:39]: yeah there're probably alot of these kinds of ts typings laying around from older code ill hunt through these commit specific ones

File: src/selectors/blockchain.ts (hunk 2 above)
Comment (W3stside) [reply src/selectors/blockchain.ts]: achso ok ill take a look and edit these, cheers

**Pull request merged**